package.json

```diff
@@ -141,7 +141,7 @@
     "@trunkio/launcher": "^1.3.0",
     "@types/node": "^20.11.5",
     "@typescript-eslint/parser": "^8.4.0",
-    "@wdio/browser-runner": "^9.0.5",
+    "@wdio/browser-runner": "9.2.1",
```

Comment:
`@wdio/browser-runner@9.2.2` is currently producing an error during `npm install` (#19064):

```console
npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "workspace:": workspace:*
npm error A complete log of this run can be found in: .../.npm/_logs/2024-10-28T08_15_15_704Z-debug-0.log
```

Reply:
I pushed a fix in `[v9.2.3](https://github.com/webdriverio/webdriverio/commit/b6de71bc75df8ea9e8a0b1dfe79ee7a4a4df21cc)` for it.